data/korath/korath missions.txt

```diff
@@ -257,3 +257,53 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Korath Boarding Response"
+	invisible
+	boarding
+	repeat
+	minor
+	deadline 1
+	to offer
+		not "Korath Boarding Response: active"
+		or
+			and
+				random < 5
+				"reputation: Korath" < -10
+				"reputation: Korath" >= -100
+			and
+				random < 10
+				"reputation: Korath" < -100
+				"reputation: Korath" >= -1000
+			and
+				random < 25
+				"reputation: Korath" < -1000
+				"reputation: Korath" >= -10000
+			and
+				random < 50
+				"reputation: Korath" < -10000
```

Comment:
idk if I can put this on multiple lines, dont think parser will understand that but here is what I meant:
```suggestion
				random < (5 * ("reputation: Korath" < -10)) + (5 * ("reputation: Korath" < -100)) + (15 * ("reputation: Korath" < -1000)) + (25 * ("reputation: Korath" < -10000))
```
this is equivalent to your long thing
is it clearer? idk, no need to apply if you think its not, I'm just showing what I think we can do

same can be done for the civilians ofc, and all could even be grouped in a mega condition
if we can put this on multiple lines it could even be clearer, but idk if we can

Reply:
That is definitely harder to read.